pkgs/development/tools/imatix_gsl/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, pcre } :
+
+stdenv.mkDerivation rec {
+    name = "imatix_gsl";
+    version = "4.1";
+    src = fetchgit {
```

Comment:
Yes, please. Or you could push each fixup as you go and squash before merge. Up to you.